test/EFCore.PG.FunctionalTests/Query/FullTextSearchDbFunctionsNpgsqlTest.cs

```diff
@@ -859,6 +859,49 @@ public void Complex_query()
             Assert.Equal("<b>Accounting</b> <b>Manager</b>", headline);
         }
 
+        [Fact]
+        public void Unaccent()
+        {
+            using var context = CreateContext();
+            var _ = context.Customers
+                .Select(x => EF.Functions.Unaccent(x.ContactName))
+                .FirstOrDefault();
+
+            AssertSql(@"SELECT unaccent(c.""ContactName"")
+FROM ""Customers"" AS c
+LIMIT 1");
+        }
+
+        [Fact]
+        public void Unaccent_with_constant_regdictionary()
+        {
+            using var context = CreateContext();
+            var _ = context.Customers
+                .Select(x => EF.Functions.Unaccent("unaccent", x.ContactName))
+                .FirstOrDefault();
+
+            AssertSql(@"SELECT unaccent('unaccent', c.""ContactName"")
+FROM ""Customers"" AS c
+LIMIT 1");
+        }
+
+        [Fact]
+        public void Unaccent_with_parameter_regdictionary()
```

Comment:
you were right, having it as a parameter caused some errors. So I did basically what was done for the Regconfig by adding Regdictionary TypeMapping and the right translation. The only thing the the NpgsqlDbType doesn't contain the regdictionary value. So for now I put Oid, let me know if it's ok or if I should do a PR in the Npgsql repo to add the Regdictionary to NpgsqlDbType

Reply:
I'm a bit surprised this the mapping works with just NpgsqlDbType.Oid - that means that for OID types, PG does allow implicit casting/coercion. But I won't argue with the passing tests :) I'm definitely fine with doing it this way.

In fact, this means we don't need dedicated regconfig/regdictionary type mappings - I'll do that bit of cleanup after merging your PR.

Reply:
On second thought, we need dedicated type mappings because of the literal string handling part.